scripts/create-ssh-config.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/usr/env bash
+
+set -e
+cd $(dirname "$0")
+
+if [[ "${SSH_CONFIG_DIR}" == "" ]]; then
+  echo Please specify the SSH_CONFIG_DIR environment variable
+  exit 1
+fi
+
+if [[ "${SSH_CONFIG_FILE}" == "" ]]; then
+  echo Please specify the SSH_CONFIG_FILE environment variable
+  exit 1
+fi
+
+INFRA_NIMBUS_SSH_CONFIG="${SSH_CONFIG_DIR}/${SSH_CONFIG_FILE}"
+: ${SSH_USERNAME:=$(whoami)}
+
+mkdir -p "${SSH_CONFIG_DIR}" && chmod 700 "${SSH_CONFIG_DIR}"
+rm -f "${INFRA_NIMBUS_SSH_CONFIG}"
+
+for host in $(ansible all -i ../ansible/inventory/test --list-hosts | grep -v 'hosts')
+do
+	cat << EOF >> "${INFRA_NIMBUS_SSH_CONFIG}"
+Host nimbus-$host
+  Hostname $host.statusim.net
```

Comment:
is there any case where one wouldn't simply want `*.statusim.net`?

Reply:
The point of having separate SSH config file entries for me is that this allows the shell auto-completion scripts to suggest the long hostnames when you type `ssh` commands. For more details, see the original description in the PR.